### PR TITLE
Lmh issues merge

### PR DESCRIPTION
**Pull request by LZRS**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ name: Android CI with Gradle
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, lmh_issue_fixes_vts_2.0 ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, lmh_issue_fixes_vts_2 ]
```
Comment (Rkareko) [inline .github/workflows/ci.yml:10]: Will this change be merged to master?

File: opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java [hunk 2]
```diff
@@ -376,19 +377,36 @@ private void registerDrawer(Activity parentActivity) {
...
     }
 
     private void registerLogout(final Activity parentActivity) {
         mPresenter.displayCurrentUser();
-        tvLogout.setOnClickListener(v -> logout(parentActivity));
+        AlertDialog logOutDialog = menuFlavor.doLogOutDialog(parentActivity);
+        tvLogout.setOnClickListener(v -> {
+//            drawer.closeDrawers();
+            if (logOutDialog != null) {
+                logOutDialog.setButton(DialogInterface.BUTTON_POSITIVE, "Log Out", (dialog, which) -> logout(parentActivity));
```
Comment (Rkareko) [inline opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java:403]: Move `Log Out` to strings.xml

File: opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java [hunk 3]
```diff
@@ -376,19 +377,36 @@ private void registerDrawer(Activity parentActivity) {
...
+                logOutDialog.setButton(DialogInterface.BUTTON_NEGATIVE, "Cancel", (dialog, which) -> dialog.dismiss());
```
Comment (Rkareko) [inline opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java:404]: Move `Cancel` to strings.xml

File: opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java [hunk 4]
```diff
@@ -431,28 +449,31 @@ private void registerLanguageSwitcher(final Activity context) {
...
-
-                // destroy current instance
-                drawer.closeDrawers();
-                instance = null;
-                Intent intent = context.getIntent();
-                context.finish();
-                context.startActivity(intent);
-                application.notifyAppContextChange();
+        if (menuFlavor.hasMultipleLanguages()) {
+            rlIconLang.setOnClickListener(v -> {
```
Comment (Rkareko) [inline opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java:453]: Could `rlIconLang` be renamed to something more descriptive

File: opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java [hunk 5]
```diff
@@ -624,12 +647,18 @@ public void setSelectedView(String selectedView) {
...
         Intent getStockReportIntent(Activity activity);
 
         Intent getServiceReportIntent(Activity activity);
 
         String childNavigationMenuCountString();
 
         Intent getHIA2ReportActivityIntent(Activity activity);
+
+        default AlertDialog doLogOutDialog(Activity activity) {
+            return null;
```
Comment (Rkareko) [inline opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java:661]: Why is this returning null ?

Comment (LZRS) [reply opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java:661]: By default it was set to null, to not show any log out dialog...but could be extended in inheriting classes for a custom pop up

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**